e2e/_suites/fleet/fleet.go

```diff
@@ -1004,46 +1025,6 @@ func createDefaultHTTPRequest(url string) curl.HTTPRequest {
 	}
 }
 
-// createFleetPolicy() sends a POST request to Fleet creating a new test policy
```

Comment:
We are not creating the policy because of https://github.com/elastic/kibana/pull/81236